packages/graphiql/package.json

```diff
@@ -31,9 +31,10 @@
   "scripts": {
     "webpack": "webpack --config resources/webpack.config.js",
     "analyze-bundle": "cross-env NODE_ENV=production CDN=1 ANALYZE=1 yarn webpack -p",
-    "build": "yarn build-clean && yarn build-js && yarn build-flow",
+    "build": "yarn build-clean && yarn build-cjs && yarn build-esm && yarn build-flow",
     "build-clean": "rimraf esm dist",
-    "build-js": "babel src --ignore __tests__ --out-dir dist/ && cross-env ESM=1 babel src --ignore __tests__ --out-dir esm/",
+    "build-cjs": "babel src --ignore __tests__ --out-dir dist/ && mkdirp dist/css && copy 'src/css/*.css' dist/css",
```

Comment:
here i have to copy files to the dist/esm outputs so modules will resolve. this doesn't feel right, however it looks like it leads to a working webpack build, which is what our users want most besides CDN build

Reply:
Can we move the css back out of the `src` folder like it was previously? This'd mean the path to it would be the same from within `src/`, `dist/` and `esm/` folders.

Reply:
i wish we could but webpack doesn't make that easy now. I will look into this. either way, its a temporary solution until we replace source css files (will still have bundled graphiql.css of course, just no source .css files) so i dont want to fuss with it too much